src/containers/MenuContainer.js

```diff
@@ -0,0 +1,17 @@
+import Menu from '../components/Menu.js';
+import { connect } from 'react-redux';
+import actions from "../actions";
+
+const mapStateToProps = (state) => {
+  return {}; // TODO - complete
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    handleEvent: (event) => {
```

Comment:
Is renaming `initaiteConversation` to `handleEvent` helpful? I expect the first is a better name..